tests/k2-components/test_async_builtins.php

```diff
@@ -0,0 +1,31 @@
+<?php
```

Comment:
Add here `@ok` to count it as a test

Reply:
k2-component tests do not belong to any test system yet. As you may see, none of existing component tests have that tag